src/Function.cpp

```diff
@@ -77,6 +77,59 @@ static long cur_func_idx;				// Index into FuncList that we are currently workin
 static bool param_first=true;			// Flag to track output of commas
 static int builtin_functions_cnt;
 
+FunctionAttribute::FunctionAttribute(string name, int prob)
+	:  attribute(name), attribute_probability(prob)
+{
+	//Nothing else to do
+}
+
+void
+FunctionAttribute::OutputAttribute(std::ostream &out, string option)
```

Comment:
Please make a default value of `option = ""` so that callers don't have to call it with `""`.